app/Http/Controllers/BookController.php

```diff
@@ -17,10 +17,19 @@ public function __construct(Book $book)
         $this->book=$book;
     }
 
-    public function getAllBooks()
+    public function getAllBooks(Request $request)
     {
         $books = $this->book->all();
-
+        
```

Comment:
You should add some validation at this point - as per the docs there should be some validation on the claimed field

```
"claimed": [
            "The claimed field must be a number."
        ],
```

I would suggest the following:

*   must be a number
*   can be between 0 and 1

Reply:
Validation added in